### ISSUE TITLE
GLM single file converter is not working

### ISSUE THREAD

**Issue by aivanova5** [arras-energy/gridlabd-old#242]
Running the command with ieee123 
~~~
   host% gridlabd -D glm_save_options=MINIMAL model/ieee123.glm test.glm
~~~

gives Segmentation fault error: 
~~~
/Users/alyona_slac/gridlabd/macosx/install/bin/gridlabd: line 360: 68765 Segmentation fault: 11  "$bindir/gridlabd.bin" "$@"
~~~

**Comment by dchassin** [comment]
The problem is that write method is an error and now all errors cause an exit.  You can either set the ignore_errors global or we can change this so it is a warning instead of an error.

**Comment by aivanova5** [comment]
It looks like I was using the wrong variable (I updated the Wiki pages from `gld_save_options` to `glm_save_options` - typo?). The comment above is updated `glm_save_options` gives a set fault. 

Ignore errors wasn't merged into master yet... Could that still be related? @dchassin

**Comment by aivanova5** [comment]
Addressed by #252
